AUTHORS

```diff
@@ -25,4 +26,4 @@ Phoenix Déb
 Rafael Audibert
 Steffen Grundke
 WL
-wolffshots
+wolffshots
```

Comment:
We want a newline at the end of this file. Besides that we are ready to merge!

Reply:
oh okay I'll add that too

Reply:
Done with this